src/property.hpp

```diff
@@ -21,9 +21,11 @@
 
 #include "util/tagged_bool.hpp"
 
+#include <string>
```

Comment:
This include was previously in the correct place.

Reply:
I'm not entirely sure why, but If this is not here I get: 

```
In file included from /Users/cm/Realm/realm-object-store/src/property.hpp:24:
/Users/cm/Realm/realm-object-store/CMakeFiles/realm-core-5.0.1/include/realm/data_type.hpp:27:9: error: unknown type name 'int64_t'
typedef int64_t Int;
```

Reply:
data_types.hpp needs to include <cstdint>